metatheory/Main.lagda

```diff
@@ -50,20 +54,58 @@ postulate
 {-# FOREIGN GHC import qualified Data.Text as T #-}
 {-# COMPILE GHC putStrLn = Text.putStrLn #-}
 
+-- IO Stuff
+
 postulate
-  return : ∀ {a} {A : Set a} → A → IO A
-  _>>=_  : ∀ {a b} {A : Set a} {B : Set b} → IO A → (A → IO B) → IO B
+  returnIO : ∀ {a} {A : Set a} → A → IO A
+  _>>=IO_  : ∀ {a b} {A : Set a} {B : Set b} → IO A → (A → IO B) → IO B
```

Comment:
Why not use [`IO.Primitive`](https://github.com/agda/agda-stdlib/blob/master/src/IO/Primitive.agda)?

Reply:
good idea, thanks. I will also look at whether to refactor other things to use stuff from there.

Reply:
Decided against using the file operations/`putStrLn` from `IO.Primitive` as it used the 'musical' colists and I am using stuff from `bytestring` anyway.